File: Modul 3/Soal 4/PRAK304-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>
int main () {

    int nilai;
    printf("Masukkan Nilai :");
    scanf("%d", &nilai);
    if (nilai==0) {
        printf("Nol");
    }
    else if (nilai >= 1 && nilai <9) {
        printf("Satuan");
    }
    else if (nilai==10 || nilai>=20 && nilai<=99) {
        printf("puluhan");
    }
    else if (nilai>=11 && nilai<=19) {
        printf("Belasan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}

File: Modul 3/Soal 4/PRAK304-2410817220018-ANGGRAENI DWI ZAHRA.py
Nilai = int(input("Masukkan Nilai = "))
if Nilai == 0:
    print("Nol")
elif Nilai >= 1 and Nilai <= 9:
    print ("Satuan")
elif Nilai == 10 or Nilai >= 20 and Nilai <= 99:
    print ("Puluhan")
elif Nilai >= 11 and Nilai <= 19:
    print ("Belasan")
else:
    print ("Anda Menginput Melebihi Limit Bilangan")
